Add new client LiveFin to clients.ts

kpgamingz opened the pull request:
Rerequesting the client I created. After the first time I requested, I fixed the looming issues with a website and broader release.

kpgamingz committed:
Update clients.ts

Rerequesting

nielsvanvelzen reviewed (changes requested):
Please add a proper title/description to the pull request. From a quick glance it looks like this client uses the Jellyfin logo which is against our guidelines.

kpgamingz commented:
The app was updated today to comply with the branding guidelines.

kpgamingz commented:
Wil be updated. 

@nielsvanvelzen ?

kpgamingz commented:
> Please put the name of the client you are adding in the title of the PR

Done.

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz commented:
Is there anything else needed for the pull request to be completed?

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz commented:
Any updates?

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz commented on src/data/clients.ts:
```diff
@@ -1027,6 +1027,53 @@ const thirdPartyClients: Array<Client> = [
+    ]
+  },
+  {
+    id: 'jellyfin-xbox',
```
It wasn't on purpose. It's added when concurrent commits are merged into the file.

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

BotBlake reviewed (changes requested):
Why are you re-requesting my review without doing any changes?
If this PR would be merged in this state, jellyfin-xbox would be in the list twice.

kpgamingz committed:
Remove Jellyfin for Xbox client entry

Removed Jellyfin for Xbox client details from the clients list.

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

BotBlake commented:
Please stop merging master into this PR!
It is only useful if there are merge conflicts.

Otherwise it keeps pinging maintainers for no reason and they will end up disabling notifications for this PR.

Kind regards.

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Merge branch 'master' into patch-1

kpgamingz committed:
Fix trailing commas

JPVenson commented on src/data/clients.ts:
```diff
@@ -984,7 +984,6 @@ const thirdPartyClients: Array<Client> = [
       {
         id: 'app-store',
         name: 'App Store',
-        url: 'https://apps.apple.com/us/app/jellify/id6736884612'
```
what about this?

kpgamingz committed:
Readded missing URL

thornbill reviewed (changes requested):
We are in the process of refining our strategy regarding the client list. Until this is ready we will not be adding any new clients.

kpgamingz committed:
Update client license and links

Changed the license type from Proprietary to Open Source and added the GitHub repository, per the upcoming rules.
